internal/support/scheduler/utils.go

```diff
@@ -104,11 +106,12 @@ func parseInt64(value string) int64 {
 // Scheduler Queue Client
 var currentQueueClient *QueueClient // Singleton used so that queueClient can use it to de-reference readings
 type QueueClient struct {
+	loggingClient logger.LoggingClient
 }
```

Comment:
(for new GitHub issue) QueueClient receiver should be moved to schedule.go where it's methods reside.